### ISSUE TITLE
Narrower font widths

### ISSUE THREAD

**Issue by cayhorstmann** [githubnext/monaspace#22]
Could the width axis go to smaller values than 100? Narrow fonts are useful to show longer lines in a limited space. https://horstmann.com/unblog/2010-11-22/fonts.html

**Comment by curiousdannii** [comment]
I've been using Iosevka for it's very narrow variants, so having more narrow font choices would be great.

See also https://github.com/be5invis/Iosevka/issues/2081

**Comment by kizu** [comment]
Agree: the minimal width of the fonts seems to be larger than widths of other fonts, making it harder to Monaspace as a drop-in replacement, as it increases the overall text column width. Would be nice to have some lower width values available.

**Comment by aclarknexient** [comment]
I'd very much like to use a condensed/narrow version of these new fonts!

I currently use [MPlus Code](https://mplusfonts.github.io/) which unfortunately doesn't support ligatures, and I'm intrigued by Monaspace and the potential for more typesetting while writing code.

For further condensed examples, look at:

* https://rubjo.github.io/victor-mono/
* https://www.programmingfonts.org/#iosevka
* https://fonts.google.com/specimen/Inconsolata/tester <-- choose 50, 62.5, or 75% width
* https://www.programmingfonts.org/#quinze
* https://fonts.google.com/specimen/Nanum+Gothic+Coding

Personally, I'd like to see Monaspace fonts support 50, 62.5, and 75% widths.

Thank you for releasing them, good fonts are very difficult to get right!

**Comment by cdluminate** [comment]
+1. The font is too wide. There is not a "Thin" variant, but a "SemiWide" and a "Wide" variant.

**Comment by Igonato** [comment]
For me Consolas font has the perfect width. Would be nice if the smallest width variant reached at least that narrowness.

<img width="682" alt="image" src="https://github.com/githubnext/monaspace/assets/788609/99a35602-5882-427a-bc58-0e28553fd7d1">

**Comment by kikones34** [comment]
I would love to use these fonts, but this is holding me back. We use a 120 character line limit for our codebase, and with these fonts at minimum width the code no longer fits into my screen :(

**Comment by NuckChorris** [comment]
*Extremely* excited to see this has been added to a Project and Milestone! Looking forward to the 1.3 release, hoping to switch from Iosevka when it drops 💃

**Comment by cdluminate** [comment]
> _Extremely_ excited to see this has been added to a Project and Milestone! Looking forward to the 1.3 release, hoping to switch from Iosevka when it drops 💃

+1. It is a necessary feature to implement before I can use this font in my terminal for a sensible information density.

**Comment by griwes** [comment]
This being moved to "someday" kinda makes me want to entirely stop tracking development of this font. You've got good ideas in it, but this is an absolute blocker for many people.

**Comment by hybridherbst** [comment]
I'd also love to see a more condensed version of this font!

**Comment by chtenb** [comment]
Please stop spamming notifications with comments that add nothing

**Comment by zhenlige** [comment]
Notice that 0.5em-wide fonts are extremely useful for CJK users, since Hanzi (Chinese ideographs) are typically 1em wide. Also, Courier is 0.6em wide, which is nearly the widest in all commonly-used monospaced fonts, and is already considered too wide occasionally, but Monaspace is 0.62em wide, EVEN WIDER THAN THAT.

**Comment by jothefiredragon** [comment]
@zhenlige Hi, I think I found a hack that might be able to slightly mitigate the issue with CJK characters. I could be wrong about any of the details though so please correct me if I'm incorrect. Full comment of mine can be found here: https://github.com/githubnext/monaspace/issues/59#issuecomment-4295460726

TL;DR of that long comment, Use [FontFreeze](https://mutsuntsai.github.io/fontfreeze/) to “Freeze” Monaspace font with the width set to 115 (percent). The “Frozen” Monaspace font can be used in the ratio of 5:7 (7 Monaspace glyphs should occupy the same amount of horizontal space as 5 CJK glyphs)
